app/code/local/Magmodules/Webwinkelconnect/Model/Observer.php

```diff
@@ -164,26 +164,60 @@ public function sendSyncUrl(): void {
             return;
         }
 
-        $curl = new Varien_Http_Adapter_Curl();
-        $curl->setConfig([
-            'timeout'   => 10,
-        ]);
         $url = 'https://dashboard.webwinkelkeur.nl/webshops/sync_url';
         $data = json_encode([
             'webshop_id' => $helper->getShopId(),
             'api_key' => $helper->getApiKey(),
             'url' => Mage::getBaseUrl(Mage_Core_Model_Store::URL_TYPE_WEB) . 'webwinkelkeur/index/sync',
         ]);
+
+        $ch = $helper->getCurlHandle($url, [
+            CURLOPT_HTTPHEADER => ['Content-Type: application/json'],
+            CURLOPT_POST => true,
```

Comment:
You [don't need](https://curl.se/libcurl/c/CURLOPT_POSTFIELDS.html) `CURLOPT_POST` when specifying `CURLOPT_POSTFIELDS`.